cmd/process-agent/main_common.go

```diff
@@ -94,7 +94,13 @@ func getSettingsClient(_ *cobra.Command, _ []string) (settings.Client, error) {
 
 	httpClient := apiutil.GetClient(false)
 	ipcAddress, err := ddconfig.GetIPCAddress()
-	ipcAddressWithPort := fmt.Sprintf("http://%s:%d/config", ipcAddress, ddconfig.Datadog.GetInt("process_config.cmd_port"))
+
+	port := ddconfig.Datadog.GetInt("process_config.cmd_port")
+	if port <= 0 {
+		return nil, fmt.Errorf("invalid process_config.cmd_port -- %d", port)
```

Comment:
The `./process-agent config` command doesn't log any message so I opted to keep throwing the error here to make it clear to the user that they're using a wrong value.

Reply:
Oh, nice catch! I totally forgot about that bit